staging/src/k8s.io/apiserver/pkg/server/options/server_run_options.go

```diff
@@ -337,5 +345,15 @@ func (s *ServerRunOptions) AddUniversalFlags(fs *pflag.FlagSet) {
 		"This option, if set, represents the maximum amount of grace period the apiserver will wait "+
 		"for active watch request(s) to drain during the graceful server shutdown window.")
 
+	utilfeature.DefaultMutableVersionedFeatureGate.DeferErrorsToValidation(true)
 	utilfeature.DefaultMutableFeatureGate.AddFlag(fs)
+	serverversion.Effective.AddFlags(fs)
```

Comment:
If my project embeds k8s.io/apiserver (like many controllers that also serve secured metrics for example) and my version numbers are not kube version numbers, what will happen if someone sets the `--emulation-version` that this is registering to 1.1 (my project's version number) which vendors kube 1.32?